doc/api/errors.md

```diff
@@ -250,10 +250,12 @@ not capture any frames.
 
 #### error.message
 
-Returns the string description of error as set by calling `new Error(message)`.
+The `error.message` property is the string description of error as set by calling
+`new Error(message)`.
 The `message` passed to the constructor will also appear in the first line of
 the stack trace of the `Error`, however changing this property after the
-`Error` object is created *may not* change the first line of the stack trace.
+`Error` object is created *may not* change the first line of the stack trace
+(for example, when `error.stack` is set before this property is changed).
```

Comment:
I thought "called" was good at first. But it is good for not property but function.
"read" is good?